lib/index.js

```diff
@@ -216,3 +222,26 @@ function _checkAccount({req, id}) {
       });
   }
 }
+
+async function _verifyTurnstileToken({token, remoteip}) {
+  const cfg = config['account-http'];
+  const {SECRET_KEY, url} = cfg.turnstile;
+
+  const payload = {
+    secret: SECRET_KEY,
+    response: token,
+    remoteip
+  };
+
+  try {
+    const result = await httpClient.post(url, {json: payload});
+    if(result.data.success) {
+      // captcha was successful, do nothing
+      return;
+    } else {
+      throw new Error('Unable to complete request, please try again.');
+    }
```

Comment:
Don't need an `else` with an early return ... but we'll want to make this a `NotAllowedError` `BedrockError` as well.

```suggestion
    }
    throw new Error('Unable to complete request, please try again.');
```